### PR TITLE
tests: update mysql-tester commit

### PR DESCRIPTION
**Pull request by Defined2014**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary: Update mysql-tester commit. See https://github.com/pingcap/mysql-tester/pull/109 to get details.

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  >   

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: tests/integrationtest/r/executor/cluster_table.result [hunk 1]
```diff
@@ -45,10 +45,10 @@ id	v
 rollback;
 select tidb_decode_sql_digests('["e6f07d43b5c21db0fbb9a31feac2dc599787763393dd5acbfad80e247eb02ad5","58f3717da2d79c14773a1e3094aaddeff2b11747d3aef95741151af9acba9d44","7e9f826ed22a09940940a42d1aca47a75b3adc3c3fde252f4b912ac886194eb9"]', 0);
 tidb_decode_sql_digests('["e6f07d43b5c21db0fbb9a31feac2dc599787763393dd5acbfad80e247eb02ad5","58f3717da2d79c14773a1e3094aaddeff2b11747d3aef95741151af9acba9d44","7e9f826ed22a09940940a42d1aca47a75b3adc3c3fde252f4b912ac886194eb9"]', 0)
-[null,"select @@tidb_current_ts","select `id` , `v` from `test_func_decode_sql_digests` where `id` = ? for update"]
+["begin","select @@tidb_current_ts","select `id` , `v` from `test_func_decode_sql_digests` where `id` = ? for update"]
```
Comment (Defined2014) [inline tests/integrationtest/r/executor/cluster_table.result:48]: A bug before.

File: tests/integrationtest/r/planner/core/plan_cache.result [hunk 2]
```diff
@@ -1842,13 +1832,11 @@ update t set a=2 where a<12;
...
-update t set a=1 where a<10;
- update t set a=2 where a<12;
-update t set a=1 where a<10;
- update t set a=2 where a<12;
+update t set a=1 where a<10; update t set a=2 where a<12;
+update t set a=1 where a<10; update t set a=2 where a<12;
 select @@last_plan_from_cache;
 @@last_plan_from_cache
-1
+0
```
Comment (Defined2014) [inline tests/integrationtest/r/planner/core/plan_cache.result:1839]: Also a bug before.

File: tests/integrationtest/t/expression/issues.test [hunk 3]
```diff
@@ -1918,3 +1918,16 @@ SELECT @@GLOBAL.information_schema_stats_expiry;
 SELECT @@GLOBAL.information_schema_stats_expiry;
 set @@SESSION.information_schema_stats_expiry=default;
 set @@GLOBAL.information_schema_stats_expiry=default;
+
+# TestIssue11333
```
Comment (Defined2014) [inline tests/integrationtest/t/expression/issues.test:1922]: Not support this test before mysql-tester update.

File: tests/integrationtest/t/expression/misc.test [hunk 4]
```diff
@@ -581,3 +581,33 @@ set @@tidb_enable_vectorized_expression = false;
...
+## current db is empty
+-- error 1046
+SELECT xxx(1);
+-- error 1046
+SELECT yyy();
+-- error 1305
+SELECT T.upper(1);
+
+## current db is not empty
+use test;
```
Comment (SeaRise) [inline tests/integrationtest/t/expression/misc.test:597]:
```suggestion
```

Comment (Defined2014) [reply tests/integrationtest/t/expression/misc.test:597]: It's by design. See L586, we didn't call `use databse` before. It's for test different error msg.

Comment (SeaRise) [reply tests/integrationtest/t/expression/misc.test:597]: ok

**Review by SeaRise — APPROVED** [review]

**Review by tiancaiamao — APPROVED** [review]

**Review by XuHuaiyu — APPROVED** [review]

**Pull request merged**